docs/semgrep-appsec-platform/dashboard-beta.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: dashboard-beta
+title: Dashboard (beta)
+hide_title: true
+description: Use the Semgrep dashboard (beta) to gain an overview of your organization's security posture, including the deployment of guardrails.
+tags:
+  - Semgrep AppSec Platform
+  - Secure guardrails
+---
+
+# Dashboard (beta)
+
+The Semgrep dashboard (beta) is an overview of your organization’s security posture from data aggregated within Semgrep AppSec Platform. It helps you achieve the following goals:
+
+- Evaluation of the success of your AppSec program, enabling you to know your current security risk.
+  - This includes assessing the deployment of secure guardrails to your organization.
+- Awareness of trends and opportunities that you can use to improve your security posture.
+  - You are able to view **recurring security issues**, consequently taking action on them.
+
+<!--
+The dashboard provides the following features:
+
+- Filters that enable you to track key metrics over a period of time, per Semgrep product, and per project.
+-->
+
+:::note
+This feature is in a **private beta**. To request access, contact your Technical Account Manager or your Account Executive and let them know you'd like to join the dashboard beta.
+:::
+
+
+![Dashboard (beta) page](/img/dashboard-fold.png)
+_**Figure**. The dashboard (beta_) page.
+
+The dashboard is divided into several panes:
+
+<dl>
+<dt>Filters and configuration</dt>
```

Comment:
i wonder if we want to call out that you can view metrics for a single product, or choose to view metrics for all products? the experience differs slightly, in that when you're viewing a single product, you won't be able to group by product. this may be a nuance we don't want to call out, but just wanted to flag it!

Reply:
Added!
![image](https://github.com/user-attachments/assets/d3d48481-a93e-4e76-9e38-be042e310d5e)